Frontend/Controllers/Projects/BenefitsController.cs

```diff
@@ -132,6 +132,16 @@ public async Task<IActionResult> OtherFactors(string urn)
             {
                 Project = project.Result,
             };
+            
+            if (otherFactors.Count == 0)
+            {
+                var firstOtherFactor =
+                    EnumHelpers<TransferBenefits.OtherFactor>.GetDisplayableValues(TransferBenefits.OtherFactor
+                        .Empty)[0];
+                model.FormErrors.AddError(firstOtherFactor.ToString(), "otherFactors",
+                    "Please select at least one other factor");
+                return View(model);
+            }
```

Comment:
See my previous comment for concerns with this - but here i'd expect to allow other factors being 0, but ensuring the descriptions are filled in when selected

This makes it a bit harder to add new ones cos it'll be tied to each field but, not much we can do about that at the moment unless you can find a nice way to put a dictionary into a view (i couldn't at the time and needed to move on)